### PR TITLE
Add part 2 blog post ingestion testing

### PR DESCRIPTION
**Pull request by JavierLainparty**
Add blog post

### CODE REVIEW

**Comment by taras** [comment]
@JavierLainparty I haven't had a chance to read it yet, but in the mean time - can you please find a way to make this image smaller 
image


@cowboyd we can generate SVGs from terminal output. I think we need to do this to make it easier to see Jest output.

**Comment by cowboyd** [comment]
@taras Yeah, I'll look into recording those as SVG animations. It might actually add a bit of pep to the post, looks like https://github.com/nbedos/termtosvg is good (although no longer maintained)

@JavierLainparty Any chance you can update your GitHub profile to contain your avatar? You're so much more than a space invader :) 👾 


<blockquote><div> GitHub</div><div><strong><a href="https://github.com/nbedos/termtosvg">GitHub - nbedos/termtosvg: Record terminal sessions as SVG animations</a></strong></div><div>Record terminal sessions as SVG animations. Contribute to nbedos/termtosvg development by creating an account on GitHub.</div></blockquote>

File: src/blog/2022-04-26-ingestion-tests-in-backstage-part-2.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+In this article we’ll demonstrate this by refactoring to upgrade a Backstage server from the legacy LDAP org processor to the new LDAP entity provider, all without changing a single line of the test case.
+
+In this article we’ll demonstrate this by refactoring in order to upgrade a Backstage server from the legacy LDAP org processor to the new LDAP entity provider – all without changing a single line of code in the test case.
```
Comment (taras) [inline src/blog/2022-04-26-ingestion-tests-in-backstage-part-2.md]: This is duplicated.

**Review by taras — CHANGES_REQUESTED** [review]
We need to fix the image size and improve the terminal output.

**Comment by taras** [comment]
> @taras Yeah, I'll look into recording those as SVG animations. It might actually add a bit of pep to the post, looks like https://github.com/nbedos/termtosvg is good (although no longer maintained)

That would be good

**Comment by cowboyd** [comment]
@JavierLainparty Here is the raw asciinema cast of the tests running https://asciinema.org/a/491054

**Comment by taras** [comment]
@JavierLainparty let me know if you'd like help getting this file embedded.

**Comment by taras** [comment]
@cowboyd what about something like this to generate HTML from the terminal? https://github.com/masukomi/oho

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/masukomi/oho">GitHub - masukomi/oho: oho: colorful terminal output to html converter</a></strong></div><div>oho: colorful terminal output to html converter. Contribute to masukomi/oho development by creating an account on GitHub.</div></blockquote>

**Comment by cowboyd** [comment]
@taras I couldn't get that tool to work on my `m1` Mac. I can try compiling from source.

```
dyld[81365]: Library not loaded: /opt/homebrew/Cellar/pcre/8.45/lib/libpcre.1.dylib
  Referenced from: /opt/homebrew/Cellar/oho/1.3.5/bin/oho
```

**Comment by taras** [comment]
Oh right, that sucks

**Comment by cowboyd** [comment]
Looks like building from source is a no-go either as it throws an exception..... feels like this is harder than it should be

**Comment by cowboyd** [comment]
Another option would be to use this in playwright to render the terminal, and then get back the HTML https://github.com/xtermjs/xterm.js/ 🤷🏻 

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/xtermjs/xterm.js">GitHub - xtermjs/xterm.js: A terminal for the web</a></strong></div><div>A terminal for the web. Contribute to xtermjs/xterm.js development by creating an account on GitHub.</div></blockquote>

**Comment by cowboyd** [comment]
Ok, we'll see if this works. I was able to install this, and capture the raw terminal output from Jest in both cases.

 https://github.com/buildkite/terminal-to-html


Here is the gist that contains `pass.html` and `fail.html` along with the raw terminal output that generated them.

https://gist.github.com/cowboyd/204fe1947fe9868a2d18a2fe1fb2cfd9


<blockquote><div> GitHub</div><div><strong><a href="https://github.com/buildkite/terminal-to-html">GitHub - buildkite/terminal-to-html: Converts arbitrary shell output (with ANSI) into beautifully rendered HTML</a></strong></div><div>Converts arbitrary shell output (with ANSI) into beautifully rendered HTML - GitHub - buildkite/terminal-to-html: Converts arbitrary shell output (with ANSI) into beautifully rendered HTML</div></blockquote>

**Comment by cowboyd** [comment]
@JavierLainparty We can either use an `iframe` on this, or we can try and massage the html and only take the elements / styles that we need.

**Comment by cowboyd** [comment]
Superseded by #223

**Pull request closed without merging**